PlanMyTrip/PlanMyTrip/SelectServicesViewController.m

```diff
@@ -16,16 +25,40 @@ @implementation SelectServicesViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+    _hotelsB.backgroundColor = [UIColor whiteColor];
+    _flightsB.backgroundColor = [UIColor whiteColor];
+    _carsB.backgroundColor = [UIColor whiteColor];
 }
 
-/*
-#pragma mark - Navigation
 
-// In a storyboard-based application, you will often want to do a little preparation before navigation
-- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
-    // Get the new view controller using [segue destinationViewController].
-    // Pass the selected object to the new view controller.
+
+- (IBAction)hotelsCheck:(id)sender {
```

Comment:
Also, I mentioned last time these buttons are all duplicated. We can create a new custom button that encapsulates these logic/setup instead of writing the same function 3 times in a row.